internal/catalogmanager/manager.go

```diff
@@ -489,7 +500,29 @@ func (m catalogManager) ProcessIntegrationVersion(version types.IntegrationVersi
 	if err := endpoints.GenerateIntegrationVersionChangelogEndpoint(m.config.StagingDir, integration, version, changelog); err != nil {
 		return fmt.Errorf("error generating integration version changelog endpoint: %w", err)
 	}
-	// TODO(jk): add directory for images or extra files
+
+	// iterate through each .jpg file in the img directory and create an
+	// endpoint for it
+	imgPath := path.Join(integrationPath, "img")
+	files, err := ioutil.ReadDir(imgPath)
+	if err != nil {
+		return fmt.Errorf("error reading integration img directory: %w", err)
+	}
+
+	for _, f := range files {
+		if !f.IsDir() {
+			match, _ := regexp.MatchString(`.*\.(jpg|gif|png)$`, f.Name())
```

Comment:
Do we have access to something like file magic? I'm wondering if it would be better for validation.

Reply:
There seems to be some Go modules that provide that kind of functionality but I'd like to cover that in a separate issue as it should be used in other parts of the project as well.

Reply:
https://github.com/sensu/catalog-api/issues/15